spirit/category/admin/tests.py

```diff
@@ -89,6 +90,16 @@ def test_category_form_color(self):
 
         self.assertFalse(form.is_valid())
 
+    def test_category_move(self):
+        utils.login(self)
+        response = self.client.get(
+            reverse('spirit:admin:category:move_dn', kwargs={"category_id": self.category.pk, }))
+        expected_url = reverse("spirit:admin:category:index")
+        self.assertRedirects(response, expected_url, status_code=302)
+        response = self.client.get(
+            reverse('spirit:admin:category:move_up', kwargs={"category_id": self.category.pk, }))
+        expected_url = reverse("spirit:admin:category:index")
+        self.assertRedirects(response, expected_url, status_code=302)
```

Comment:
You have to tests those API behaviour (i.e their side effects) not just what response is returned.